src/test/skript/junit/EvtExperienceCooldownChangeTest.sk

```diff
@@ -0,0 +1,16 @@
+on load:
+	set {_test::1} to "got player"
+	set {_test::2} to "got reason"
+	set {_test::3} to "got new cooldown"
+
+	ensure junit test "org.skriptlang.skript.test.tests.syntaxes.events.EvtExperienceCooldownChangeTest" completes {_test::*}
+
+on player experience cooldown change:
+	junit test is "org.skriptlang.skript.test.tests.syntaxes.events.EvtExperienceCooldownChangeTest"
```

Comment:
you should set the test string to a variable to avoid massive strings everywhere